src/test/regress/bin/normalize.sed

```diff
@@ -14,9 +14,6 @@ s/shard [0-9]+/shard xxxxx/g
 s/assigned task [0-9]+ to node/assigned task to node/
 s/node group [12] (but|does)/node group \1/
 
-# discard "USING heap" in "CREATE TABLE ... USING heap"
-s/CREATE(.*)TABLE(.*)USING heap/CREATE\1TABLE\2/g
```

Comment:
FYI: This almost certainly stems from the days where we supported PG11, because table access methods were not a thing back then. And probably PG12 would output this USING part, but PG11 would not.

Reply:
Thanks, that makes sense!